feconf.py

```diff
@@ -51,9 +51,13 @@
 TESTS_DATA_DIR = os.path.join('core', 'tests', 'data')
 SAMPLE_EXPLORATIONS_DIR = os.path.join('data', 'explorations')
 SAMPLE_COLLECTIONS_DIR = os.path.join('data', 'collections')
-INTERACTIONS_DIR = os.path.join('extensions', 'interactions')
-GADGETS_DIR = os.path.join('extensions', 'gadgets')
-RTE_EXTENSIONS_DIR = os.path.join('extensions', 'rich_text_components')
+
+EXTENSIONS_DIR_PREFIX = 'build_staging' if (IS_MINIFIED or not DEV_MODE) else ''
```

Comment:
I borrowed the concept from _templates/dev/head_ and _templates/prod/head_ because the files are built into the _prod_ folder and then copied into the _build_ folder. The backend uses HTML files from _prod_ and frontend from _build_. I do the same but with extensions. If used this way the name _build_staging_ is probably not appropriate.